### ISSUE TITLE
count function support column parameter

### ISSUE THREAD

**Issue by gemron** [influxdata/influxdb-client-java#149]
like this :
```
from(bucket: "example-bucket")
  |> range(start: -5m)
  |> count(column: "_value")
```

**Comment by bednar** [comment]
Hi @gemron,

thanks for using our client.

Do you mean in` flux-dsl`? Something like this:

```java
Flux flux = Flux
     .from("telegraf")
      .count("_value");
```

Regards

**Comment by gemron** [comment]
@bednar Thank you for your prompt reply.  
yes ,it is in flux-dsl.
In addition, I think the limit method can support the offset parameter.
like this:
```
Flux flux = Flux
     .from("telegraf")
     .limit(100,10);
```
first parameter is offset,second is limit number.

**Comment by bednar** [comment]
Thanks for detail info - We will prepare fix in PR: #150

**Comment by bednar** [comment]
Hi @gemron,

The issue is fixed in [1.12.0 milestone](https://github.com/influxdata/influxdb-client-java/milestone/15). 

If you would like to use a preview version use our snapshot build from repository: https://oss.sonatype.org/content/repositories/snapshots/

Regards

**Comment by gemron** [comment]
@bednar Okay, thank you for your efforts. Let me try it.
